pkg/controllers/secretstore/glob_test.go

```diff
@@ -0,0 +1,77 @@
+/*
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package secretstore
```

Comment:
We shouldn't test private functions directly. Would you add tests to `TestManagerGet` in client_manager_test.go?

Reply:
I did not find any tests on conditions on that test. However I added the test in `pkg/controllers/externalsecret/externalsecret_controller_test.go`. Really good framework. It was easy to implement :)